Makefile

```diff
@@ -5,12 +5,59 @@ SUDO := $(shell docker version >/dev/null 2>&1 || echo "sudo")
 LABEL ?= $(shell git rev-parse --short HEAD)$(and $(shell git status -s),-dirty-$(shell id -u -n))
 # If TAG is not provided set default value
 TAG ?= stellar/anchor-platform:$(LABEL)
+E2E_TAG ?= stellar/anchor-platform-e2e-test:$(LABEL)
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 BUILD_DATE := $(shell date -u +%FT%TZ)
+DOCKER_COMPOSE_TEST_CONFIG_DIR = integration-tests/docker-compose-configs
+DOCKER_COMPOSE_TEST_BASE_FILE := $(DOCKER_COMPOSE_TEST_CONFIG_DIR)/docker-compose.base.yaml
+DOCKER_COMPOSE_TEST_OVERRIDE_FILE = docker-compose-config.override.yaml
 
 docker-build:
 	$(SUDO) docker build -f Dockerfile --pull --label org.opencontainers.image.created="$(BUILD_DATE)" \
 	-t $(TAG) .
 
 docker-push:
 	$(SUDO) docker push $(TAG)
+
+docker-build-e2e-test:
+	$(SUDO) docker build -f end-to-end-tests/Dockerfile --pull --label org.opencontainers.image.created="$(BUILD_DATE)" \
+	-t $(E2E_TAG) .
+
+docker-push-e2e-test:
+	$(SUDO) docker push $(E2E_TAG)
+
+build-docker-compose-tests:
+	docker-compose -f integration-tests/docker-compose-configs/docker-compose.base.yaml build --no-cache
+
+run-e2e-test-all:
+	make run-e2e-test-default-config
+	make run-e2e-test-allowlist
+	make run-e2e-test-unique-address
```

Comment:
In my test, the second routine failed and the third one did not get called, which is the expected behavior 👍.

The only thing I missed is a way to quickly identify which one of these failed. Do you know of a way to easily do so? I'm not very experienced with Dockerfile/Makefile myself, so I'm not sure either, but it'd be helpful to debug what went wrong.

If it's too troublesome, we can move forward without that. When we get to Jenkins though, we should probably have one step for each test case. This way, we can identify from GH which test was broken.

Reply:
added the -p parameter to the docker-compose command which will include the project name when the test exits
![image](https://user-images.githubusercontent.com/11511032/186982094-47990a3b-3987-4233-b171-5cd8967df1e5.png)